30_get-pipenv

```diff
@@ -52,32 +52,49 @@ install_pipenv()
   # add it to the python path, and install virtualenv and it's distlib
   # dependency, that will now have the freshly installed setuptools in the
   # PYTHONPATH
-  "${PIPENV_PYTHON}" "${get_pip}" --no-cache-dir -I --root "${TMP_DIR}" pip
+  "${PIPENV_PYTHON}" "${get_pip}" --no-cache-dir -I --prefix "${TMP_DIR}/1" pip
 
   # In order to get python to use our custom root dir, we must set the PYTHONPATH to its
-  # site-packages directory. Unfortunately, when we ask site for the answer, it gives
+  # site-packages directory. Unfortunately, when we to ask site for the answer, it gives
   # multiple answers. Just use them all
-  SITE_PACKAGES="$("${PIPENV_PYTHON}" -c "if True:
-          import os, site
-          print(':'.join([os.path.join('${TMP_DIR}',x.lstrip(os.path.sep))
-                  for x in site.getsitepackages()]))")"
+  # For some off reason this broke. Replacing with prefix instead of root, and
+  # using os.walk (aka find) instead of using site.
+  # SITE_PACKAGES="$("${PIPENV_PYTHON}" -c "if True:
+  #         import os, site
+  #         print(':'.join([os.path.join('${TMP_DIR}',x.lstrip(os.path.sep))
+  #                 for x in site.getsitepackages()]))")"
+
+  # Find the directory containing the pip dir, that's site-packages
+  PIP_SITE_PACKAGES="$("${PIPENV_PYTHON}" -c "if True:
+          import os
+          for root, dirs, files in os.walk('${TMP_DIR}/1'):
+            if 'pip' in dirs:
+              print(root)
+              break")"
```

Comment:
Will this error find if there is no directory containing `pip`, or continue on its merry way?  If you're not guaranteed an error, maybe check that `PIP_SITE_PACKAGES` is not empty.